### ISSUE TITLE
Thunderbird 10 and searching the web

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2118]
**Reported by challsworth2 on 2012-02-21 11:48**
One of the new features in Thunderbird 10 is to search the web based on what you have selected to search for in a message. This appears as a right click menu item. Unfortunately this does not work when browse mode is active, however it does when focus mode is active.

**Comment by leonardder** [comment]
Confirmed. However, This is likely to require updates to the virtual buffer system, which is probably not going to happen.

**Comment by Adriani90** [comment]
@challsworth2, @leonardder, can you still reproduce this issue?

**Comment by Adriani90** [comment]
Testing with NVDA 2019.1.1 and Thunderbird 60.6.1, this issue is still reproducible. It seems you can select a i.e. link in browse mode with shift+arrow keys which is reported by NVDA. However, the context menu looks differently if you select the same link while you are in focus mode. In focus mode, NVDA does not report anything if you press shift+arrow keys to select the link. But When you press context menu, there is an entry called "bing search for <link>".

In conclusion, it seems the behabior when selecting a link in browse mode seems to be separated from the behavior when selecting the same link in focus mode.

**Comment by Adriani90** [comment]
This is no longer a feature in Thunderbird 68.7, at least not from the context menu. Both context menus in browse and focus mode are now identical. Closing as invalid.
